### ISSUE TITLE
[FR] Better control over learning rate and hyperparameter settings inside swyft network

### ISSUE THREAD

**Issue by james-alvey-42** [PEREGRINE-GW/peregrine#13]
**Is your feature request related to a problem? Please describe.**
Custom learning rate modules e.g. linear warm up etc. should be able to be passed to swyft network via the `configure_callbacks` function

**Describe the solution you'd like**
Examples and config loading to achieve this
